SpotifyWebApi/Model/SimplePlaylist.cs

```diff
@@ -63,12 +63,11 @@ public class SimplePlaylist
         [JsonProperty("snapshot_id")]
         public string SnapshotId { get; set; }
 
-        // TODO: This causes error, fix this.
-        // /// <summary>
-        // /// Gets or sets the tracks.
-        // /// </summary>
-        // [JsonProperty("tracks")]
-        // public List<LinkedFrom> Tracks { get; set; } = new List<LinkedFrom>();
+        /// <summary>
+        /// Gets or sets the tracks.
+        /// </summary>
+        [JsonProperty("tracks")]
+        public LinkedFrom Tracks { get; set; }
```

Comment:
After looking at the docs a bit it seems to me we just need a `Paging<Tracks>` here.
https://developer.spotify.com/documentation/web-api/reference/#/operations/get-playlist

```suggestion
        public Paging<SimpleTrack> Tracks { get; set; }
```